ref/network-ports.md

```diff
@@ -10,7 +10,6 @@ For the Anbox Cloud Appliance, ports are exposed only for accessing the Anbox Cl
 | Service               | Port(s)     | Protocol  | Exposed externally | Required                     | Description                           |
 |-----------------------|-------------|-----------|--------------------|------------------------------|---------------------------------------|
 | AMS                   | 8444        | TCP       | no                 | yes                          | HTTPS API                             |
-| AMS                   | 20002       | TCP       | no                 | no                           | HTTPS Prometheus endpoint             |
```

Comment:
I removed this because it is offering a prometheus endpoint and we don't talk about it anywhere else. Does that make sense?

Reply:
Reverted the change as clarified that we offer the endpoint even if we don't offer a monitoring solution so that customers can build their custom monitoring solution.